content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,23 @@ menu:
     parent: About the project
 ---
 
+## v1.8.6 [2021-05-21]
+
+{{% note %}}
+**Fine-grained authorization security update.** If you're on InfluxDB 1.8.5, we recommend immediately upgrading to this release. An issue was reported in 1.8.5 where grants with specified permissions for users were not enforced. Versions prior to InfluxDB 1.8.5 are not affected. This security update ensures that only users with sufficient permissions can read and write to a measurement.
```

Comment:
InfluxDB Enterprise 1.8.5 -- add the Enterprsie in both places where InfluxDB is used.

Reply:
thanks, @timhallinflux!